arex-instrumentation/dubbo/arex-dubbo-alibaba/src/main/java/io/arex/inst/dubbo/alibaba/DubboAdapter.java

```diff
@@ -146,4 +150,24 @@ public String getProtocol() {
     public String getConfigVersion() {
         return invocation.getAttachment(ArexConstants.CONFIG_VERSION);
     }
+
+    @Override
+    protected Map<String, String> getRequestHeaders() {
+        Map<String, String> headerMap = RpcContext.getContext().getAttachments();
+        headerMap.put(KEY_PROTOCOL, getProtocol());
+        headerMap.put(KEY_GROUP, getValByKey(DubboConstants.KEY_GROUP));
+        headerMap.put(KEY_VERSION, getValByKey(DubboConstants.KEY_VERSION));
+        return headerMap;
+    }
+
+    private String getValByKey(String key) {
```

Comment:
This method appears three times in different classes. Can it be extracted and called as a common method?

Reply:
`Invocation/Invoker/URL` different version, different package